generators/app/templates/client/app/_app.core.module.js

```diff
@@ -31,6 +31,8 @@
     $locationProvider.html5Mode(true).hashPrefix('!');
   }
 
+  configure.$inject = ['$locationProvider'];
```

Comment:
You don't need to provide this so long as we're using a good ng-annotate script along with the `@ngInject` annotation. I've purposely avoided this to keep the code cleaner. See this for more details: https://github.com/olov/ng-annotate#explicit-annotations-with-nginject